### PR TITLE
Add the personal info screen in the account app.

### PR DESCRIPTION
**Pull request by gerbermichi**
## Summary

Enables the personal info screen section in the account app.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   keycloak:
-    image: quay.io/keycloak/keycloak:15.0.0
+    image: jboss/keycloak:15.0.2
```
Comment (chloesoe) [inline docker-compose.yml:4]: Is this correct? For some reason we had quay.io I guess

Comment (chloesoe) [reply docker-compose.yml:4]: But it's only for the local spin up of the theme, therefore it is ok change it.

Comment (chloesoe) [reply docker-compose.yml]: changed it back

Comment (gerbermichi) [reply docker-compose.yml]: ok, thx.

**Pull request merged**